assets/black-friday-scripts.js

```diff
@@ -33,15 +33,21 @@ $(document).ready(function() {
   });
 
   $('.apply_discount').on('click', function() {
-    var discount = $(this).prev('input[name="discount_code"]').val();
+    var discount = $(this).siblings('input[name="discount_code"]').val();
     var value = $(this).data('value');
     const collectionHref = $(this).data('collectionHref')
 
-    if (discount != '') {
-      document.cookie = 'discount_code=' + discount + '; expires=0; path=/';
-
+    document.getElementById('discountInput').onkeyup = function(event) {
+      if ( !this.value.length ) {
+        document.getElementById('textOfError').style.visibility = "hidden";
```

Comment:
@PrudinnikDmitriy formatting please
```suggestion
        document.getElementById('textOfError').style.visibility = 'hidden';
```

Reply:
@yevheniitsuryk  fixed